src/NuGet.Core/NuGet.PackageManagement/Utility/XmlUtility.cs

```diff
@@ -1,125 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.IO;
-using System.Xml;
-using System.Xml.Linq;
-
-namespace NuGet.ProjectManagement
-{
-    public static class XmlUtility
```

Comment:
Just pointing out that this is an API breaking change.

Personally I don't care because NuGet is not an XML processing library, so I don't think customers should be depending on us to provide this functionality. If someone disagrees, then we'll need to bring this class back and make it a wrapper for the internal class.

Reply:
I'm more concerned about the different NuGet assemblies interplay. 
Given that this is not xplat, it's a lot less likely to happen.